packages/auth/src/authClients/azureActiveDirectory.ts

```diff
@@ -0,0 +1,29 @@
+import { UserAgentApplication as AzureActiveDirectory } from 'msal'
+
+export type { AzureActiveDirectory }
+import type { AuthClient } from './'
+
+export type AzureActiveDirectoryClient = AzureActiveDirectory
+export interface AzureActiveDirectoryUser {}
+
+export const azureActiveDirectory = (client: AzureActiveDirectoryClient): AuthClient => {
+  return {
+    type: 'azureActiveDirectory',
+    client,
+    restoreAuthState: async () => {},
+    login: async (options?) => {
+      await client.loginPopup(options)
+    },
```

Comment:
```suggestion
    login: async (options?) => await client.loginPopup(options),
```

Reply:
Fixed in https://github.com/jeliasson/redwood/commit/5b2caee1228e8c6579d0f202f691041188fe1720